app/src/utils/accessControl.ts

```diff
@@ -47,18 +54,18 @@ export const requireCanViewProject = async (projectId: string, ctx: TRPCContext)
         },
       });
     } else {
-      throw new TRPCError({ code: "UNAUTHORIZED" });
+      throw new TRPCError({
+        code: "UNAUTHORIZED",
+        message: "You don't have access to view this project.",
+      });
     }
   }
 };
 
 export const requireCanModifyProject = async (projectId: string, ctx: TRPCContext) => {
   ctx.markAccessControlRun();
 
-  const userId = ctx.session?.user.id;
-  if (!userId || !projectId) {
```

Comment:
This check is actually important. If userId or projectId are not set, then the prisma.projectUser.findFirst call will error. Please add it back in to all the access check functions.